runtime/src/storage/mkvs/urkel/tree/lookup.rs

```diff
@@ -49,6 +49,11 @@ impl UrkelTree {
         let boxed_key = key.to_vec();
         let pending_root = self.cache.borrow().get_pending_root();
 
+        // If the key has been modified locally, no need to perform any lookups.
+        if let Some(PendingLogEntry { ref value, .. }) = self.pending_write_log.get(&boxed_key) {
```

Comment:
Only for serialization/deserialization. This is because write logs are only ever applied at the server side which is in Go. This commit adds a simple protocol test (the new `test_syncer_writelog_remove`) that makes sure write log formats are compatible.

Reply:
No, I plan to port the test fixture runner that we use in Go to Rust and use the same test cases here.

Reply:
Done.